packages/components/psammead-most-read/src/List/index.test.jsx

```diff
@@ -52,7 +52,7 @@ describe('MostReadList', () => {
 
   shouldMatchSnapshot(
     'should render with ltr news items with max two columns',
-    <MostReadList numberOfItems={10}>
+    <MostReadList numberOfItems={10} columnLayout="twoColumn">
```

Comment:
Should we also have snapshots for `columnLayout="multiColumn"` and `columnLayout="oneColumn"` ?